lib/Mojolicious/Command/version.pm

```diff
@@ -0,0 +1,111 @@
+package Mojolicious::Command::version;
+use Mojo::Base 'Mojolicious::Command';
+
+use Mojo::IOLoop::Client;
+use Mojo::IOLoop::TLS;
+use Mojo::JSON;
+use Mojolicious;
+
+has description => 'Show versions of available modules';
+has usage       => sub { shift->extract_usage };
+
+sub run {
+  my $self = shift;
+
+  my $json  = Mojo::JSON->JSON_XS                   ? $Cpanel::JSON::XS::VERSION   : 'n/a';
+  my $ev    = eval { require Mojo::Reactor::EV; 1 } ? $EV::VERSION                 : 'n/a';
+  my $socks = Mojo::IOLoop::Client->can_socks       ? $IO::Socket::Socks::VERSION  : 'n/a';
+  my $tls   = Mojo::IOLoop::TLS->can_tls            ? $IO::Socket::SSL::VERSION    : 'n/a';
+  my $nnr   = Mojo::IOLoop::Client->can_nnr         ? $Net::DNS::Native::VERSION   : 'n/a';
+  my $roles = Mojo::Base->ROLES                     ? $Role::Tiny::VERSION         : 'n/a';
+  my $async = Mojo::Base->ASYNC                     ? $Future::AsyncAwait::VERSION : 'n/a';
+
+  print <<EOF;
+CORE
+  Perl        ($^V, $^O)
+  Mojolicious ($Mojolicious::VERSION, $Mojolicious::CODENAME)
+
+OPTIONAL
```

Comment:
While optional, are they *recommended*?  i.e. If you have the ability to install them -- do it without question?  Or might there be a reason to not use the optional module if not needed?